tests/data_files/pkcs7-rsa-sha256-1.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Do these PEM-formatted files need to be PEM-formatted? If not, DER is preferable: PEM files can't be used for tests in a configuration without PEM, and they're harder to inspect manually when debugging.

Reply:
Yes, I would like to have test run with PEM formatted files as that is how it is getting used in actual application. I would add MBEDTLS_PEM_C as dependency as required.

Reply:
But many others run builds without PEM support (PEM is rarely used in the embedded world since the text encoding has a significant overhead at that scale). [We're trying to maximize the test coverage without PEM support](https://github.com/ARMmbed/mbedtls/issues/2402). Except when specifically testing support for PEM-encoded data, binary data gives you as much coverage, and has the advantage that builds without PEM get the same coverage.